backend/app/assets/javascripts/spree/backend/templates/orders/line_item.hbs

```diff
@@ -4,7 +4,11 @@
   </td>
 {{else}}
   <td class="line-item-image">
-    <img src="{{ image.mini_url }}">
+    {{#if image }}
+      <img src="{{ image.mini_url }}">
+    {{else}}
+      <span class="image-placeholder mini"></span>
+    {{/if}}
```

Comment:
It can be done, but I think it warrants further discussion. I will open a follow-up PR.